frontend/awx/administration/instances/InstanceDetails.tsx

```diff
@@ -54,16 +55,27 @@ export function InstanceDetails() {
     useInstanceActions(params.id as string);
   const pageNavigate = usePageNavigate();
   const postRequest = usePostRequest();
+  const activeUser = useAwxActiveUser();
+  const { data } = useGet<Settings>(awxAPI`/settings/system/`);
+
+  const canAddAndEditInstances =
```

Comment:
There are different requirements for adding vs editing an instance. We need to check for node type as well.